docs/elements/reference/common.mdx

```diff
@@ -0,0 +1,444 @@
+<SignIn.Root>
+  <Clerk.GlobalError>
+    {({ message, code }) => (
+      <span>
+        {message} ({code})
+      </span>
+    )}
+  </Clerk.GlobalError>
+</SignIn.Root>
+```
+
+## `<Icon>`
+
+`<Icon>` **must** be used inside [`<Connection>`](#connection). By default, `<Icon>` will render as an `<img>` element with the `src` pointing to the logo of the currently used `<Connection>`.
+
+<Callout type="info">
+
+The purpose of `<Icon>` is to provide quick access to Clerk's social connections logos which you can use while setting up your application. Most likely you wish to customize their look or use your own icon library. Hence the scope of `<Icon>` is small and customizability limited by design. If you need further control, please consider using solutions like [Iconify](https://iconify.design/getting-started/).
+
+</Callout>
+
+### Properties {{ toc: false }}
+
+| Name       | Type                                                          | Description                                                                      |
+| ---------- | ------------------------------------------------------------- | -------------------------------------------------------------------------------- |
+| `asChild` | `boolean` (optional) | If `true`, `<Icon>` will render as its child element, passing along any necessary props. Default: `false` |
+
+### Examples {{ toc: false }}
+
+#### Basic
+
+`<Icon>` will render as the Google logo here.
+
+```tsx filename="page.tsx" {4}
+<SignIn.Root>
+  <SignIn.Step name="start">
+    <Clerk.Connection name="google">
+      <Clerk.Icon />
+      Sign in with Google
+    </Clerk.Connection>
+  </SignIn.Step>
+</SignIn.Root>
+```
+
+## `<Input>`
+
+Handles rendering of `<input />` elements within Clerk's flows. Supports special `type` prop values to render input types that are unique to authentication and user management flows. Additional props will be passed through to the [`<input />` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
+
+### Properties {{ toc: false }}
+
+| Name    | Type                                                  | Description                                                                                                                                           |
+| ------- | ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `asChild` | `boolean` (optional)                                  | If true, `<Input>` will render as its child element, passing along any necessary props. Default: `false`                                                             |
+| `name`    | `string` (optional)                                   | Name for form field, will be used to associate all underlying field elements. Can be automatically inferred from the name on the `<Field>` component. Default: Automatically inferred |
```

Comment:
```suggestion
| `name`    | `string` (optional)                                   | Name for the form field. Will be used to associate all underlying field elements. Can be automatically inferred from the name on the `<Field>` component. Default: `"Automatically inferred"` |
```

Is `"Automatically inferred"` the default string value here?

Reply:
It is not, per the sentence before, the default value is automatically inferred based on the field name.

Reply:
Ohhh, I see now. The value of the `name` prop on the `Field` component